docs/Administrator-Guide/Tuning-Volume-Options.md

```diff
@@ -0,0 +1,100 @@
+ | cluster.rebal-throttle | Sets the maximum number of parallel file migrations allowed on a node during the rebalance operation. The default value is normal and allows a max of [(((processingunits) − 4) / 2), 2] files to be migrated at a time. Lazy will allow only one file to be migrated at a time and aggressive will allow maxof[(((processing units) - 4) / 2), 4] | normal | lazy/normal/aggressive
+ | cluster.background-self-heal-count | Specifies the number of per client self-heal jobs that can perform parallel heals in the background. | 8 | 0-256
+ | cluster.heal-timeout | Time interval for checking the need to self-heal in self-heal-daemon | 600 | 5-(signed-int)
+ | cluster.eager-lock | If eager-lock is off, locks release immediately after file operations complete, improving performance for some operations, but reducing access efficiency | on | on/off
+ | cluster.quorum-type | If value is “fixed” only allow writes if quorum-count bricks are present. If value is “auto” only allow writes if more than half of bricks, or exactly half including the first brick, are present | none | none/auto/fixed
+ | cluster.quorum-count | If quorum-type is “fixed” only allow writes if this many bricks are present. Other quorum types will OVERWRITE this value | null | 1-(signed-int)
+ | cluster.heal-wait-queue-length | Specifies the number of heals that can be queued for the parallel background self heal jobs. | 128 | 0-10000
+ | cluster.favorite-child-policy | Specifies which policy can be used to automatically resolve split-brains without user intervention. “size” picks the file with the biggest size as the source. “ctime” and “mtime” pick the file with the latest ctime and mtime respectively as the source. “majority” picks a file with identical mtime and size in more than half the number of bricks in the replica. | none | none/size/ctime/mtime/majority
+ | cluster.use-anonymous-inode | Setting this option heals directory renames efficiently | no | no/yes
+Logging | diagnostics.brick-log-level | Changes the log-level of the bricks | INFO | DEBUG/WARNING/ERROR/CRITICAL/NONE/TRACE
+ | diagnostics.client-log-level | Changes the log-level of the clients. | INFO | DEBUG/WARNING/ERROR/CRITICAL/NONE/TRACE
+ | diagnostics.latency-measurement | Statistics related to the latency of each operation would be tracked. | Off | On/Off
+ | diagnostics.dump-fd-stats | Statistics related to file-operations would be tracked. | Off | On/Off
+Performance | *features.trash | Enable/disable trash translator | off | on/off
+ | *performance.readdir-ahead | Enable/disable readdir-ahead translator in the volume |	off | on/off
+ | *performance.read-ahead | Enable/disable read-ahead translator in the volume | off | on/off
+ | *performance.io-cache | Enable/disable io-cache translator in the volume | off | on/off
```

Comment:
Please remove the prefixed star

Reply:
These prefix are intentionally added. This is highlighted for below note in line100

Reply:
Cool sounds good.